realm/realm-library/src/objectServer/java/io/realm/internal/network/LogoutResponse.java

```diff
@@ -79,16 +81,6 @@ private LogoutResponse(String serverResponse) {
      * @return {@code true} if valid.
      */
     public boolean isValid() {
-//        return (error == null);
-        return true;
-    }
-
-    /**
-     * Returns the error.
-     *
-     * @return the error.
-     */
-    public ObjectServerError getError() {
-        return error;
+        return (error == null) || (error.getErrorCode() == ErrorCode.EXPIRED_REFRESH_TOKEN);
```

Comment:
This doesn't look right? `(error.getErrorCode() == ErrorCode.EXPIRED_REFRESH_TOKEN)` should (error.getErrorCode() != ErrorCode.EXPIRED_REFRESH_TOKEN)` ?